The-Recipe-Manager-App/src/application/CardController.java

```diff
@@ -150,11 +169,37 @@ public void sendObj(Recipe recipe) {
 			}
         });
     }
+    public void sendObject(Recipe recipe) { 
+    	System.out.println("Inside card Controller"); 
+    	saveBtn.setOnAction(event -> { 
+   		System.out.println("Entering Save Button");
+   			// Create a new SavedRecipesController instance 
+   			FXMLLoader loader = new FXMLLoader(getClass().getResource("SavedRecipes.fxml"));
+   			try {
+        		Parent root = loader.load();
+				SavedRecipesController controller = loader.getController();
+	        	
+	        	if(SavedRecipesController.recipes.contains(recipe)) {
+	        		Alert alert = new Alert(AlertType.INFORMATION);
+	                alert.setTitle("Recipe Saved");
+	                alert.setHeaderText(null);
+	                alert.setContentText("The recipe is already saved.");
+	                alert.showAndWait();
+	        	}
+	        	else {
+	        		controller.handleSaveButtonClick(recipe);
+		        	System.out.println("Object recieved at cardController" + recipe);
+		        	SavedRecipesController.recipes.add(recipe);
+
+	        	}
+        	} catch (IOException e) {
+				e.printStackTrace();
+        	}
+            
+   		});
+    }
 
-    
-   public void MoreBtnAction(ActionEvent e) {
-	   
-   }
+ 
```

Comment:
check where is MoreBtnAction

Reply:
ActionEvent for more button is in another method called sendObj. So that one was not needed.